File: Janhavi_Tripathi/day2.cpp
#include<bits/stdc++.h>
using namespace std;                    //[3,1,2,4]
                                                             
class Solution {
public:
    vector<int> sortArrayByParity(vector<int>& nums) {
        int i=0;                                              
        for(int j=0;j<nums.size();j++)                   
        {
            if(nums[j]%2==0)                               
            {
              swap(nums[i],nums[j]);                           
            i++;                                               
            }                                      
        }
             
      return nums;
    }
};

File: anant_21gcebds053/day 2.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
              
		int i,k=0,r;
		
		int b[]= new int[nums.length];
	
       
       
        for(i=0;i<nums.length;i++)
        {
            if((nums[i]%2)==0)
            {
                b[k]=nums[i];
                k++;
            }
        }
        r=k;
         for(i=0;i<nums.length;i++)
         {
             if((nums[i]%2)!=0)
             {
                 b[r]=nums[i];
                 r++;
             }
         }
        return b;
	}
}



        
    
